torch/_refs/__init__.py

```diff
@@ -4238,8 +4240,10 @@ def tensor_split(
         indices = indices_or_sections
         if isinstance(indices_or_sections, TensorLike):
             if indices_or_sections.ndim != 1:
-                msg = "tensor_split: non-scalar indices_or_sections tensors must have only one dimension, "
-                f"but received a tensor with {indices_or_sections.ndim} dimensions"
+                msg = (
+                    "tensor_split: non-scalar indices_or_sections tensors must have only one dimension, "
+                    f"but received a tensor with {indices_or_sections.ndim} dimensions"
```

Comment:
@haampie We may want to enforce this with the relevant ruff rules: [(ISC002)](https://docs.astral.sh/ruff/rules/multi-line-implicit-string-concatenation/#multi-line-implicit-string-concatenation-isc002)

Reply:
I think you need a linter first, cause there's no string concatenation because the newline line character is not escaped. The second line is a stand-alone expression with an unused string.